src/Sylius/Bundle/CoreBundle/DependencyInjection/SyliusCoreExtension.php

```diff
@@ -56,6 +56,8 @@ public function load(array $config, ContainerBuilder $container)
         $config = $this->processConfiguration($this->getConfiguration([], $container), $config);
         $loader = new XmlFileLoader($container, new FileLocator(__DIR__.'/../Resources/config'));
 
+        $loader->load(sprintf('services/integrations/%s.xml', $config['driver']));
```

Comment:
This file is not needed. In CoreBundle it is enough to declare repository in the resource tree.